packages/react/src/custom.d.ts

```diff
@@ -0,0 +1,5 @@
+// Ambient module to allow the importing of .mdx files in storybook .tsx files
+declare module '*.mdx' {
+  const content: any;
```

Comment:
We should definitely aim for zero warnings. This should either be `unknown` or if working around the lack of explicit `any` is too onerous to devs (since this is a gradual conversion), perhaps we consider disabling the `no-explicit-any` rule to start.

Reply:
Good point, i should have changed this to `unknown`